### PR TITLE
Add MultiBooked dataset

### PR DESCRIPTION
**Pull request by albertvillanova**
Add dataset.

### CODE REVIEW

File: datasets/multi_booked/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+[More Information Needed]
```
Comment (lhoestq) [inline datasets/multi_booked/README.md:86]: could you add description of each field of the dataset please ?

File: datasets/multi_booked/multi_booked.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                            "pos": datasets.Value("string"),
+                            "target": datasets.features.Sequence(datasets.Value("string")),
+                        }
+                    ),
+                    "opinions": datasets.features.Sequence(
+                        {
+                            "oid": datasets.Value("string"),
+                            "opinion_holder_target": datasets.features.Sequence(datasets.Value("string")),
+                            "opinion_target_target": datasets.features.Sequence(datasets.Value("string")),
+                            "opinion_expression_polarity": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/multi_booked/multi_booked.py:93]: Could you use the ClassLabel feature type here ?
For example
```
"opinion_expression_polarity": datasets.ClassLabel(names=["Negative", "Positive"])
```

you'll need to regenerate the dataset_infos.json file after this change

Also note that if there are missing values, you should use the integer `-1` (currently the script returns an empty string for missing polarities)

**Review by lhoestq — COMMENTED** [review]
Really cool thank you !

I left a few comments.
One in particular is about using ClassLabel at least for the polarity field

Let me know what you think

File: datasets/multi_booked/README.md (hunk 1 above)
Comment (albertvillanova) [reply datasets/multi_booked/README.md]: @lhoestq done.

File: datasets/multi_booked/multi_booked.py (hunk 2 above)
Comment (albertvillanova) [reply datasets/multi_booked/multi_booked.py]: @lhoestq also done. There aren't missing values.

File: datasets/multi_booked/multi_booked.py [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                                # Opinion target
+                                opinion_target = child.find("opinion_target")
+                                targets = []
+                                for target in opinion_target.findall("span/target"):
+                                    targets.append(target.get("id", ""))
+                                example["opinions"]["opinion_target_target"].append(targets)
+                                # Opinion expression
+                                opinion_expression = child.find("opinion_expression")
+                                example["opinions"]["opinion_expression_polarity"].append(
+                                    opinion_expression.attrib.get("polarity", "")
```
Comment (lhoestq) [inline datasets/multi_booked/multi_booked.py:158]:
```suggestion
                                    opinion_expression.attrib["polarity"]
```
if there are no missing values, can you remove the `.get` ?
same for the other fields

Comment (albertvillanova) [reply datasets/multi_booked/multi_booked.py]: @lhoestq done!

**Review by lhoestq — APPROVED** [review]
Looks all good thank you !

**Comment by lhoestq** [comment]
There' still an issue with the dummy data, let me take a look

**Pull request merged**